tools/generator/GenBaseSupport.cs

```diff
@@ -150,6 +150,7 @@ public ManagedGenBaseSupport (TypeDefinition t)
 
 		public override string Visibility {
 			get { return t.IsPublic || t.IsNestedPublic ? "public" : "protected internal"; }
+			set { }
```

Comment:
This should throw InvalidOperationException so that anyone who works on generator should immediately notice that it is inappropriately used and not hiding the actual issue.